tests/transform.spec.ts

```diff
@@ -216,14 +230,14 @@ describe('transform tests', () => {
       '@/*': ['src/*'],
       '@components/*': ['src/components/*'],
       '@src': ['src'],
-      '*': ['src/utils/*']
+      '*': ['tests/__fixtures__/*']
     }
 
-    const aliases = parseTsAliases(resolve(__dirname), tsPaths)
+    const aliases = parseTsAliases(resolve(__dirname, '..'), tsPaths)
 
     const options = (content: string) => ({
       content,
-      filePath: resolve(__dirname, './src/index.ts'),
+      filePath: resolve(__dirname, '../src/index.ts'),
```

Comment:
aliases were assuming top level was in the `tests` folder resulting in absolute paths being `~/tests/src/...` etc. so going up a directory here fixes this